Delete chain fix

alexyoung23j opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Link to Issue
Closes: #5126 

## Description of Changes
- Adds a migration to remove a foreign key constraint that was preventing us from deleting chains that had a Discord bot connected. 

## "How We Fixed It"
<!-- Brief description of solution, if bug, to be added once issue is resolved. -->
Migration, added a cascade delete on the DiscordBotConfig table.

## Test Plan
- Attempt to delete a chain with a discord bot connected via the admin panel. 

## Deployment Plan
<!--- Omit if unneeded -->
1. 

## Other Considerations
<!--- Follow-up tickets, breaking changes, etc -->
-

jnaviask commented:
@alexyoung23j does this affect the sequelize model at all? + should we permit discord bot configs that don't satisfy this constraint? Seems like we should consider scope of impact here...

alexyoung23j commented:
Fixed model, but otherwise the scope here is pretty isolated- the update just ensures that when a chain is deleted the corresponding discord bot config is also deleted. This will always be the desired behavior afaik.

jnaviask commented on packages/commonwealth/server/models/discord_bot_config.ts:
```diff
@@ -58,6 +58,7 @@ export default (
     models.DiscordBotConfig.belongsTo(models.Chain, {
       foreignKey: 'chain_id',
       targetKey: 'id',
+      onDelete: 'CASCADE', // add this line
```
Can you remove the comment?
